src/style.css

```diff
@@ -0,0 +1,58 @@
+.bar1 {
+  width: 100%;
+  border: 1px solid black;
+  margin-top: 20px;
+  box-shadow: 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 24px;
+  background-size: 19px;
+}
+
+.bar2 {
+  width: 100%;
+  border: 1px solid black;
+  margin-top: 20px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 14px 16px;
+  background-size: 11px;
+}
+
+.input1 {
+  margin-top: 25px;
+  margin-left: 50px;
+  margin-bottom: 23px;
+  border: 0;
+  font-weight: 300;
+}
+
+.input2 {
+  margin-top: 12px;
+  margin-bottom: 10px;
+  margin-left: 33px;
+  margin-right: 0;
+  border: 0;
+  font-weight: 300;
+}
+
+.input1::placeholder {
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.input2::placeholder {
+  font-weight: 300;
+  font-size: 14px;
+}
```

Comment:
you don't need it, just add styles under `.input` selector (font-related styles, placeholder will inherit them)